test/benchgc/src/benchgc/pattern/util.py

```diff
@@ -0,0 +1,48 @@
+################################################################################
```

Comment:
These functions are generic enough. Please place it at `benchgc/util.py`
Please check function `set_md` in `benchgc/mlir/arg.py` to find a elegant way to reuse the function

Reply:
fixed, `set_md` is slightly different and can not reuse the func, so I just simplified the code for getting shape.